typescript/package.json

```diff
@@ -5,10 +5,20 @@
   "main": "index.js",
   "license": "MIT",
   "devDependencies": {
+    "@babel/core": "^7.13.8",
+    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
+    "@babel/plugin-transform-typescript": "^7.13.0",
+    "@babel/preset-env": "^7.13.9",
+    "@babel/preset-typescript": "^7.13.0",
+    "@types/jest": "^26.0.20",
+    "babel-jest": "^26.6.3",
+    "jest": "^26.6.3",
     "ts-node": "^9.1.1",
     "typescript": "^4.1.5"
   },
   "scripts": {
-    "structural:adapter": "ts-node structural/adapter/index.ts"
-  }
+    "structural:adapter": "ts-node structural/adapter/index.ts",
+    "creation:builder": "yarn jest -t 'Test New Service'"
```

Comment:
Perhaps limiting this to the specific directory too would be useful as perhaps others may want to add more tests as part of their solution. Also will make sense for `yarn jest` not have to traverse the entire set of patterns later on.

Reply:
Now it runs all tests in the ts builder directory